src/software/ai/hl/stp/play/ball_placement/ball_placement_play.cpp

```diff
@@ -17,7 +18,18 @@ void BallPlacementPlay::getNextTactics(TacticCoroutine::push_type &yield,
 
 void BallPlacementPlay::updateTactics(const PlayUpdate &play_update)
 {
-    fsm.process_event(BallPlacementPlayFSM::Update(control_params, play_update));
+    auto event = BallPlacementPlayFSM::Update(control_params, play_update);
+    fsm.process_event(event);
+
+    auto placement_point = event.common.world_ptr->gameState().getBallPlacementPoint();
```

Comment:
nit: might be better to forgo auto and use `std::optional<Point>` here since the point can be `null_opt` type which explains the necessity of the null check below.

Reply:
Done